protocol/deputy-guardian-module-improvements.md

```diff
@@ -0,0 +1,125 @@
+1. Change the respected game type to the `PermissionedDisputeGame`.
+1. Trigger the `setAnchorState` function in the `AnchorStateRegistry`.
+
+The `DeputyGuardianModule` would **no longer** be permitted to:
+
+1. Undo the Superchain-wide pause ("unpause").
+1. Change the respected game type to the `FaultDisputeGame`.
+
+The `DeputyGuardianModule` would be accessible to:
+
+1. Any member of the Security Council Safe.
+1. An EOA operated by the Optimism Foundation.
+
+Please note that this proposal supercedes
+[Design Doc 162](https://github.com/ethereum-optimism/design-docs/pull/162) that introduced the
+concept of the `DeputyPauseModule`.
+
+### Reasoning
+
+The above changes make the role of the `DeputyGuardianModule` significantly more obvious. The
+`DeputyGuardianModule` is now designed to be a fast response mechanism that can ONLY impact
+liveness and cannot impact safety.
+
+1. We remove the ability to unpause the system because this makes it possible for the
+   `DeputyGuardianModule` to diminish safety by unpausing the system when it was explicitly placed
+   into a paused state for security reasons.
+1. We restrain the `DeputyGuardianModule` to be able to change the respected game type to the
+   `PermissionedDisputeGame` but not back to the `FaultDisputeGame` for similar reasons to the
+   removal of the ability to unpause the system.
+
+## Alternatives Considered
+
+### Deputy Pause Module
+
+This proposal supercedes a previous proposal that would have installed a `DeputyPauseModule` into
+the Foundation Safe. We would be able to get the benefits of this proposal while also simplifying
+the role of the Deputy Guardian if we accepted this proposal instead.
+
+## Risks & Uncertainties
+
+### Audits
+
+Any module will need to be carefully audited. The `DeputyGuardianModule` was previously audited but
+would need to be re-audited as a result of these changes.
+
+### Leaked Deputy
+
+Our worst-case scenario is a leaked deputy private key. Since the `DeputyGuardianModule` would now
+permit any member of the Security Council Safe or a dedicated Foundation EOA to trigger
+liveness-impacting actions, such a leak would likely cause a temporary liveness failure. A majority
+of the Security Council Safe would need to coordinate a transaction to remove or replace the
+offending account and resolve the liveness failure.
+
+### Compromised Deputy
+
+A motivated attacker with access to a deputy's private key could try to drain the deputy's wallet
+constantly to prevent it from being used to trigger transactions. Although this could be
+side-stepped with private RPC/bundling tools, it would not be considered good practice to include
+live third-party infrastructure in the hot path for critical security actions. It therefore seems
+prudent that the deputy be able to act via signature instead of directly via `msg.sender` check.
+
+If we allow the deputy to act via signature then we must also make sure that the signature includes
```

Comment:
How different is this from just using 1/1 SAFEs instead of raw EOA addresses?

Reply:
If 1/1 safe is compromised then owner can be changed and you lose access to pause. Using raw EOA makes this impossible.